src/mlpack/core/cv/metrics/average_strategy.hpp

```diff
@@ -21,6 +21,13 @@ namespace cv {
  *
  * The "Binary" strategy means binary classification is going to be used, and
  * there is no need to average.
+ *
+ * The "Micro" strategy is used for multiclass classification considering
+ * TP(True Positive), TN(True Negative) and FN(False Negative) all at once
+ * for evaluating both precision and recall.
```

Comment:
This isn't clear to me and I'm not sure it's correct.  In addition, it's also specific to metrics like in the comment below.  Please update this with a correct and clear definition of what the microaveraging strategy is.